lib/Driver/FineGrainedDependencyDriverGraph.cpp

```diff
@@ -404,7 +438,8 @@ void ModuleDepGraph::emitDotFileForJob(DiagnosticEngine &diags,
 
 void ModuleDepGraph::emitDotFile(DiagnosticEngine &diags, StringRef baseName) {
   unsigned seqNo = dotFileSequenceNumber[baseName]++;
-  std::string fullName = baseName.str() + "." + std::to_string(seqNo) + ".dot";
+  std::string fullName =
+      baseName.str() + "-post-integration." + std::to_string(seqNo) + ".dot";
```

Comment:
Is this debugging code?

Reply:
Not sure what you are asking for here. The dot file facility is for debugging and visualization, as I tried to comment at the method declarations. What would be clearer? (I'm not crazy about the fact that one has to go find the declaration in order to get the scoop on the definition, but I've been told that that's how it is in this world.)